docs/fsharp-cheatsheet.md

```diff
@@ -125,33 +138,56 @@ See [Literals (MS Learn)](https://learn.microsoft.com/en-us/dotnet/fsharp/langua
 
 ## Functions
 
-The `let` keyword also defines named functions.
+Use the `let` keyword to define named functions.
 
     let negate x = x * -1
     let square x = x * x
-    let print x = printfn "The number is: %d" x
+    let print x = printfn $"The number is: {x}"
+
+### Pipe and Composition Operators
 
-### Pipe and composition operators
 Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
 
     let ``square, negate, then print`` x =
         x |> square |> negate |> print
 
-This operator can assist the F# type checker by providing type information before use:
```

Comment:
this was never the best in terms of conveying the point that sometimes using piping can let you remove the need for type annotations, but it should not be removed without covering it elsewhere

Reply:
I agree. Function pipes operators aren't covered well enough and wanted to go back to them.